packages/compass-connections/src/stores/connections-store.spec.tsx

```diff
@@ -139,11 +152,11 @@ describe('useConnections', function () {
     renderHookWithContext({ onConnected });
 
     await waitFor(() => {
-      expect(onConnected).to.have.been.called;
+      expect(onConnected).to.have.been.calledOnce;
     });
 
     // autoconnect info should never be saved
-    expect(saveSpy).to.not.have.been.called;
```

Comment:
Not sure if this was auto-replaced by accident, but we kinda expect save to not be called at all in this case, not only not called once (this will not fail if it's called twice)

Reply:
yes that was an accident 🤦 thanks!